### PR TITLE
add a vector of data and update init()

### PR DESCRIPTION
**Pull request by estarossa0**

### CODE REVIEW

**Review by ombhd — CHANGES_REQUESTED** [review]
change the getData() return type to const reference

**Comment by estarossa0** [comment]
> change the getData() return type to const reference

ok.

File: includes/Server.hpp [hunk 1]
```diff
@@ -25,5 +25,6 @@ class	Server
 	operator		int();
 	size_t			size();
 
-	ServerData		getData();
+	std::vector<ServerData const>	getData() const;
```
Comment (estarossa0) [inline includes/Server.hpp]: here

**Review by ombhd — APPROVED** [review]

File: includes/Server.hpp (hunk 1 above)
Comment (ombhd) [reply includes/Server.hpp]: add a const reference on the returned vector

**Review by ombhd — APPROVED** [review]

**Review by ombhd — APPROVED** [review]

**Pull request merged**